new/src/sections/home/home.scss

```diff
@@ -0,0 +1,68 @@
+.home {
+  position: relative;
+
+  &-left {
+    padding: 5.7vw 8vw;
+    display: flex;
+    flex-direction: column;
+    gap: 50px;
+  }
+
+  &-text {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+
+    &-lg {
+      font-size: 50px;
+      font-weight: 900;
+      line-height: 130%;
+
+      @media (min-width: 1024px) {
+        width: 42vw;
+      }
+
+      @media (max-width: 768px) {
+        font-size: 40px;
+      }
+    }
+
+    &-creative {
+      font-family: "Leckerli One", cursive;
+      font-style: italic;
+      letter-spacing: -1px;
+      color: #1471c7;
+    }
+
+    &-sm {
+      font-size: 18px;
+      line-height: 150%;
+
+      @media (min-width: 1024px) {
+        width: 29vw;
+      }
+    }
+  }
+
+  &-learn-more {
```

Comment:
For <1024px screen width, I suggest to remove the learn more button and add a learn-more hyperlink at the end of the text

![image](https://github.com/nus-gdg/nus-gdg.github.io/assets/58817781/bc4e4e9d-ca20-45f1-91e1-5c429f322ab2)

The learn more button is not immediately visible when user opens the page, user need to scroll down to find the button, in which case they wouldn't need the button any more. I think can either just remove the button, or remove it and add a hyperlink **Learn more >>** at end of the text.

Paddings can also be reduced a bit for smaller screen width